### PR TITLE
Add missing labels gha 57

### LINKED ISSUE #57: GitHub Actions: Add missing labels on issue creation
### Overview
As a developer, it would be optimal to automate parts of our workflow so mistakes are lessened. For this issue we will make an automation that adds the missing labels if one of 'role', 'feature/p-feature', or 'size' series are missing.

### Psudo-code

- [ ] When a new issue is created:
  - [ ] Read the labels that are on the issue:
  - [ ] If a 'role', 'feature/p-feature', or 'size' label is missing
    - [ ] Add the corresponding 'missing' label
    - [ ] Post an @comment to the creator of the issue reminding them to add labels if they know what labels to add

### Resources/Instructions
[Labels](https://github.com/hackforla/CivicTechJobs/labels)

Note, work on this has already been done previously at the website project. Please refer to this [file](https://github.com/hackforla/website/blob/gh-pages/github-actions/add-update-label-weekly/add-label.js).

### PR DESCRIPTION
**Pull request by bzzz-coding**
Fixes #57

### Changes

- Added a workflow job in issue-trigger.yml
- Created a `github-actions` folder to store scripts and templates
- Wrote a script that checks for required labels of a newly (re-)opened issue, add missing labels if any and post a comment requesting the issue creator to add required labels.

### CODE REVIEW

File: app/tsconfig.json
Comment (bzzz-coding) [inline app/tsconfig.json:1]: Not sure if I was supposed to add github-actions to "include"... I did this so my PR could pass the linter check 😅.

File: github-actions/issue-trigger/check-labels/add-labels-post-comment.js
Comment (bzzz-coding) [inline github-actions/issue-trigger/check-labels/add-labels-post-comment.js:1]: Should I try to write this script in TypeScript, or is it okay to use JavaScript?
Also, I'm including a lot of console logging and comments for reviewing and testing. I can add, edit or delete logging and comments if needed.

File: github-actions/issue-trigger/check-labels/comment-template.md
Comment (bzzz-coding) [inline github-actions/issue-trigger/check-labels/comment-template.md:1]: This template is copied from the [website repo](https://github.com/hackforla/website/blob/75d09777d68e4b44bafd8cc8ffc7e0a0269bb6d3/github-actions/trigger-issue/add-missing-labels-to-issues/add-labels-template.md) as of 4/15/23.

**Comment by Aveline-art** [comment]
Oh my goodness. Thanks for doing this, but we actually do not need this any more. Wow, the ticket has been written almost two years ago, and a lot of changes have happened since then! So sorry to have wasted your time. Feel free to check this work into your portfolio, though.

**Pull request closed without merging**